clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -533,6 +535,25 @@ public void testMetadataChangeTriggersRebalance() {
     }
 
     @Test
+    public void testExcludeInternalTopicsConfigOption() { 
```

Comment:
To be clear, you are subscribing to ".*" pattern (i.e. everything), and you assume there will be no assignment because the only topics that exist at this point are internal ones?

Isn't it a bit fragile? Why not subscript to TopicConstants.INTERNAL_TOPICS to be certain you won't be surprised by other tests creating topics?